predictsignauxfaibles/__main__.py

```diff
@@ -40,13 +41,30 @@ def load_conf(args):  # pylint: disable=redefined-outer-name
     return model_conf
 
 
-def run(args):  # pylint: disable=redefined-outer-name
+def evaluate(model, df):  # To be turned into a SFModel method when refactoring models
+    """
+    Returns evaluation metrics of model evaluated on df
+    Args:
+        model: a sklearn-like model with a predict method
+        df: dataset
+    """
+    balanced_accuracy = balanced_accuracy_score(df["outcome"], model.predict(df))
+    fbeta = fbeta_score(df["outcome"], model.predict(df), beta=conf.EVAL_BETA)
+    return balanced_accuracy, fbeta
```

Comment:
`return {"balanced_accuracy": balanced_accuracy, "fbeta": fbeta}`

since nothing in the function signature tells us which metrics are returned, better make it explicit

Reply:
✅